pkg/conf/config.go

```diff
@@ -36,10 +36,10 @@ func EnvConfig(appName string, conf interface{}) error {
 }
 
 // GetBuilderKey returns the key to be used as token to interact with deis-controller
-func GetBuilderKey() (string, error) {
-	builderKeyBytes, err := ioutil.ReadFile(builderKeyLocation)
+func GetBuilderKey(builderKeyPath string) (string, error) {
```

Comment:
Would it be easier to change BuilderKeyLocation to type `var` so tests can change the path before running `GetBuilderKey()`? Wondering why we'd define a constant `BuilderKeyLocation` variable, yet not use it for `GetBuilderKey()`.